CSharp/BatchExplorer/Views/MainView.xaml.cs

```diff
@@ -466,5 +466,11 @@ private void SearchJobsTxtBox_KeyUp(object sender, KeyEventArgs e)
                 Messenger.Default.Send<RefreshMessage>(new RefreshMessage(RefreshTarget.Jobs));
             }
         }
+
+        private void LinkedStorageHelp_RequestNavigate(object sender,
+                                               System.Windows.Navigation.RequestNavigateEventArgs e)
+        {
+            System.Diagnostics.Process.Start("https://docs.microsoft.com/en-us/azure/batch/batch-task-output");
```

Comment:
Based on an example - Not that I know of. It seems one is what is displayed. The other when it's clicked. WPF seems to have not great HTML support